Sources/DotNetGraph.Tests/Extensions/DotGraphExtensionsTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using DotNetGraph.SubGraph;
+using DotNetGraph.Extensions;
+using System.Collections.Generic;
+using Xunit;
+using NFluent;
+using DotNetGraph.Core;
+using DotNetGraph.Node;
+using DotNetGraph.Edge;
+
+namespace DotNetGraph.Tests.Extensions
+{
+    public class DotGraphExtensionsTests
+    {
+        public static IEnumerable<object[]> GetGraphs()
+        {
+            yield return new[] { new DotGraph("G") };
+            yield return new[] { new DotSubGraph("cluster_0") };
+        }
+
+        private void AssertFirstElement<T>(IDotGraph sut)
```

Comment:
I believe this method should be named AssertSingleElement since it asserts that elements count is 1.

Reply:
Fixed